tree_diff.go

```diff
@@ -13,13 +14,19 @@ import (
 // TreeDiff is a PipelineItem.
 type TreeDiff struct {
 	previousTree *object.Tree
+	SkipVendor   bool
 }
 
 const (
 	// DependencyTreeChanges is the name of the dependency provided by TreeDiff.
 	DependencyTreeChanges = "changes"
+	// ConfigTreeDiffSkipVendor is the name of the configuration option
+	// (TreeDiff.Configure()) which allows to skip vendor directories.
+	ConfigTreeDiffSkipVendor = "TreeDiff.SkipVendor"
 )
 
+var vendorDirs = []string{"vendor/", "vendors/", "node_modules/"}
```

Comment:
I suggest to move this list to the command line flag (comma separated). For sure there are other directories in other languages. We could name it "BlacklistedDirectories"

Reply:
sorry, could you clarify a bit?

Do you propose to add `BlacklistedDirectories` flag instead of `skip-vendor`? Or add additional flag `BlacklistedDirectories`?

The problem with replacement - it should be empty by default. So the user must know all possible vendor directories if he/she wants to skip them. Not very user-friendly 😞

Reply:
I see. Then two flags - rename vendor to blacklist and add `--backlisted-dirs`